app/gui/src/main/resources/themes/dark-theme.css

```diff
@@ -0,0 +1,34 @@
+.root {
```

Comment:
I think you should figure out how to comment in css and put a link to where you got this from in the file itself, just so we can find it later